test/e2e/capi_test.go

```diff
@@ -105,17 +105,17 @@ var _ = Describe("Running the Cluster API E2E tests", func() {
 		})
 	})
 
-	Context("Should successfully apply resources in ClusterResourceSet object to all matching clusters", func() {
-		capi_e2e.ExperimentalFeaturesSpec(context.TODO(), func() capi_e2e.ExperimentalFeaturesSpecInput {
-			return capi_e2e.ExperimentalFeaturesSpecInput{
-				E2EConfig:             e2eConfig,
-				ClusterctlConfigPath:  clusterctlConfigPath,
-				BootstrapClusterProxy: bootstrapClusterProxy,
-				ArtifactFolder:        artifactFolder,
-				SkipCleanup:           skipCleanup,
-			}
-		})
-	})
+	// Context("Should successfully apply resources in ClusterResourceSet object to all matching clusters", func() {
+	// 	capi_e2e.ExperimentalFeaturesSpec(context.TODO(), func() capi_e2e.ExperimentalFeaturesSpecInput {
+	// 		return capi_e2e.ExperimentalFeaturesSpecInput{
+	// 			E2EConfig:             e2eConfig,
+	// 			ClusterctlConfigPath:  clusterctlConfigPath,
+	// 			BootstrapClusterProxy: bootstrapClusterProxy,
+	// 			ArtifactFolder:        artifactFolder,
+	// 			SkipCleanup:           skipCleanup,
+	// 		}
+	// 	})
+	// })
```

Comment:
This test can be removed now, the cni is being installed by the resourceSet as part of the other tests, but the template used needs updating

Reply:
you also need to remove `https://github.com/kubernetes-sigs/cluster-api-provider-azure/tree/master/templates/test/prow-clusterresourceset` and `https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/test/e2e/config/azure-dev.yaml#L49-L50`
but add this https://github.com/kubernetes-sigs/cluster-api/blob/master/test/e2e/data/infrastructure-docker/cluster-template.yaml#L128-L146 to the test templates https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/templates/test/cluster-template-prow.yaml and to https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/test/e2e/data/infrastructure-azure/cluster-template-kcp-adoption.yaml